consoleme/lib/slack.py

```diff
@@ -35,6 +35,10 @@ async def send_slack_notification_new_policy_request(
     Sends a notification using specified webhook URL about a new request created
     """
 
+    if config.get("slack.ignore_auto_admin_policies", False):
```

Comment:
Missing check for whether it is actually auto-approved or not? 😄 
This will ignore *all* new policy requests if the option is set. I think this should be:
```
if admin_approved and config.get("slack.ignore_auto_admin_policies", False):
    return
```